src/scripts/main.js

```diff
@@ -1,7 +1,81 @@
 'use strict';
 
 // Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const Game = require('../modules/Game.class');
+const game = new Game();
 
 // Write your code here
+
+const gameButton = document.querySelector('.button');
+const gameScore = document.querySelector('.game-score');
+const messageWin = document.querySelector('.message-win');
+const messageLose = document.querySelector('.message-lose');
+const messageStart = document.querySelector('.message-start');
+const gameField = document.querySelector('.game-field');
+
+document.addEventListener('keyup', (arrow) => {
+  if (game.status === 'playing') {
+    switch (arrow.key) {
+      case 'ArrowUp':
+        game.moveUp();
+        break;
+      case 'ArrowDown':
+        game.moveDown();
+        break;
+      case 'ArrowRight':
+        game.moveRight();
+        break;
+      case 'ArrowLeft':
+        game.moveLeft();
+        break;
+      default:
+        break;
+    }
+
+    updateUi(game.board);
+
```

Comment:
The `updateUi` function is called after each move. Ensure that this function accurately reflects the current state of the game board and updates the UI accordingly.